source/about/frequently-asked-questions.rst

```diff
@@ -314,7 +314,7 @@ When the Mattermost Source Available ``LICENSE`` file appears in a specific dire
 
 In all cases, any third party components remain licensed under their original license.
 
-An example directory layout is shown below:
+An example directory layout, using an Enterprise Edition E20 license, is shown below:
```

Comment:
Would it make sense to change this reference to "E20" to "Edition" as well?

Reply:
I think the graphic itself also needs to be updated :) Though I recognize that some graphics are easier to update than others :) What's really great about these changes is that once they're made, we don't need to make additional updates for v6.0.

Reply:
Yeah the graphic needs to be update and there's a help wanted issue for that. @mickmister and I discussed the graphic and agreed that it can stay for now as a working example of using an E20 license in this instance. But long-term it does need to be replaced.